stock-usecase/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/MainContent.kt

```diff
@@ -81,7 +81,7 @@ fun MainContent(
     val settingsUiState by viewModel.settingsUiState.collectAsState()
     var columnHeightDp by remember { mutableStateOf(0.dp) }
     val localDensity = LocalDensity.current
-    val tablePadding = if (backdropState.isRevealed) 244.dp else 20.dp
+    val tablePadding = if (backdropState.isRevealed) 244.dp else 0.dp
```

Comment:
this will solve the issue when the editing mode is off, but should solve the gap between the keyboard and the TextInput when the edition is on?. please, let me know what do you think

Reply:
Hi, this solves this problem.

Reply:
ok!, thanks. I'll test it